src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)].toString();
```

Comment:
It's better to refactor 
```suggestion
        int index = random.nextInt(Color.values().length);
        return Color.values()[index].toString().toLowerCase();
```

[Use name() for getting String representation of enum constants](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#use-name-for-getting-string-representation-of-enum-constants)

Reply:
Not fixed